src/Illuminate/Contracts/Database/Eloquent/SupportsPartialRelations.php

```diff
@@ -7,8 +7,8 @@ interface SupportsPartialRelations
     /**
      * Indicate that the relation is a single result of a larger one-to-many relationship.
      *
-     * @param  \Closure|string|null  $column
-     * @param  string|null  $relation
+     * @param  string|null  $column
+     * @param  string|\Closure|null  $aggregate
```

Comment:
This order should not have changed.

Reply:
Okay, I guessed that types are ordered by probability. Just noticed the laravel docs show this exact example.